clients/dataset/dataset.go

```diff
@@ -10,9 +10,10 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/ONSdigital/dp-rchttp"
 	"github.com/ONSdigital/go-ns/clients/clientlog"
 	"github.com/ONSdigital/go-ns/common"
-	"github.com/ONSdigital/go-ns/rchttp"
+	"github.com/ONSdigital/go-ns/log"
```

Comment:
Is this the wrong logger? I am not sure if we can use the new logger in our go apps yet... If we can does it make sense to use PR's like this to slowly migrate across?

Reply:
it's not the new-new one (i.e it's not Ians one), it's the one we used on cmd.